client/src/app/app-routing.module.ts

```diff
@@ -2,10 +2,12 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { StockDetailComponent } from './pages/stock-detail/stock-detail.component';
 import { LeaderboardComponent } from './pages/leaderboard/leaderboard.component';
+import { LoginSignupComponent } from '../app/pages/login-signup/login-signup.component';
 
 const routes: Routes = [
   { path: 'stock-detail/:ticker', component: StockDetailComponent },
   { path: 'leaderboard', component: LeaderboardComponent },
+  { path: 'home', component: LoginSignupComponent} // TO DO: choose adequate path for landing page
```

Comment:
I discussed this point with @Alessandro100, if we choose home for our sign-in/sign-up page, it would confuse us in future, as we have a home button in our header that takes the user to the first page of the application (the one that has all the news-feed and right and left side panels, the page that user sees after signing in). If we change it now, it would be easier, I think maybe we can change the route name for this page to sing-up or welcome page.

Reply:
In that case would an empty route be the better option? 
So that the default is going to this page but if the user is already logged in we redirect to his home actual home page. 
I do not have much knowledge about internet cache so correct me if this wrong
@Alessandro100

Reply:
you are right, we will change the route to empty